config.yaml

```diff
@@ -0,0 +1,27 @@
+document: modularity-testing
```

Comment:
In addition to Jan's comment I've updated the config file to use the local image and also opened a RFE to allow us to set the default module, see: https://pagure.io/modularity-testing-framework/issue/10